src/internal_modules/roc_core/slab_pool.h

```diff
@@ -89,9 +89,9 @@ class SlabPool : public IPool, public NonCopyable<> {
                 flags) {
     }
 
-    //! Get size of objects in pool.
-    size_t object_size() const {
-        return impl_.object_size();
+    //! Get size of the allocation per object.
+    size_t allocation_size() const {
+        return SlotSize;
     }
```

Comment:
Shouldn't it be `slot_size_` ?

Reply:
Having a think... I think there's some duplication.

Reply:
OK. I've gone with your suggestion.